docs/track.rst

```diff
@@ -215,6 +222,65 @@ Example::
         }
     ]
 
+data-streams
+............
+
+The ``data-streams`` section contains a list of all data streams that are used by this track.   Cannot be used if the ``indices`` section is specified.
```

Comment:
Nit: Two spaces after the first sentence instead of one.

Reply:
3 even!